snakemake_executor_plugin_kubernetes/__init__.py

```
@@ -476,7 +512,7 @@ def cancel_jobs(self, active_jobs: List[SubmittedJobInfo]):
         # Cancel all active jobs.
         for j in active_jobs:
             self._kubernetes_retry(
-                lambda: self.safe_delete_pod(j.external_jobid, ignore_not_found=True)
+                lambda: self.safe_delete_job(j.external_jobid, ignore_not_found=True)
```

Ruff (0.8.2)

Function definition does not bind loop variable `j`

(B023)